src/main/java/cz/cvut/kbss/termit/service/mail/Postman.java

```diff
@@ -52,10 +59,20 @@ public class Postman {
 
     private final JavaMailSender mailSender;
 
-    public Postman(@Autowired(required = false) JavaMailSender mailSender) {
+    @Autowired
+    public Postman(Environment env, @Autowired(required = false) JavaMailSender mailSender) {
+        this.env = env;
         this.mailSender = mailSender;
     }
 
+    @PostConstruct
+    public void postConstruct() {
+        // exclude mailSender check for test environment
+        if(mailSender == null && Arrays.stream(env.getActiveProfiles()).noneMatch("test"::equals)) {
```

Comment:
This is ugly. Solve it by creating a mock of `Postman` in test config.

Reply:
in fact there is already a [JavaMailSender](https://github.com/kbss-cvut/termit/blob/cf5e5f35f219a53cda29ce178e44e72b2474c17e/src/test/java/cz/cvut/kbss/termit/environment/config/TestServiceConfig.java#L96), so Postman will not complain, profile check removed